base/database/batch.go

```diff
@@ -157,8 +186,9 @@ func insertFunc(scope *gorm.Scope, columnNames, groups []string) {
 		extraOptions = fmt.Sprintf(" %s", insertOption)
 	}
 
-	scope.Raw(fmt.Sprintf( //nolint:gosec
-		"INSERT INTO %s (%s) VALUES %s %s",
+	// nolint: gosec
```

Comment:
No, since we're not binding just values, but also column names. There is quoting of the values, but this is potential SQL injection vector if we ever accept unrestricted user input for column names.